src/rules_python/__init__.py

```diff
@@ -0,0 +1,8 @@
+import sys
+
+# Make it so "import python.runfiles" and "import rules_python.python.runfiles"
+# both refer to the same object. This avoids importing
+# the same code twice under different names.
+import python
+sys.modules["rules_python.python"] = python
```

Comment:
Interesting, I was more thinking of moving the code to `src/rules_python/python/runfiles`, setting bazel imports attribute correctly to import both `src` and `src/rules_python` to make it work like before, creating an alias `runfiles` library and then modifying the runfiles wheel to be built in the same way.

As a side note, if I remember correctly, if you use the Python wheel and not the bazel rules, the Python import is just `runfiles`, so the users would be doing `from runfiles import Runfiles`, or allow all three import styles?

Reply:
Yeah, I thought about moving everything under src/ but opted not to because...well, I don't really have a good reason. Mostly FUD I guess?

For lack of an actual reason not to, I'll just move everything under src/ and setup shims.